README.md

```diff
@@ -69,6 +69,27 @@ hCaptcha
         Log.d("hCaptcha", "hCaptcha is now visible.");
     }
   });
+  .addOnCloseListener(new OnCloseListener() {
+    @Override
+    public void onClose() {
+        // Usefull for analytics purposes
+        Log.d("hCaptcha", "hCaptcha is closed");
+    }
+  })
+  .addOnExpiredListener(new OnExpiredListener() {
+    @Override
+    public void onExpired() {
+        // Usefull for analytics purposes
+        Log.d("hCaptcha", "hCaptcha is expired");
```

Comment:
This would not work since the hCaptcha JS API is removed after a successful verification and thus, cannot trigger this callback.